hazelcast/src/test/java/com/hazelcast/map/impl/nearcache/NearCacheTest.java

```diff
@@ -229,6 +229,111 @@ public void testNearCacheEviction_withMapClear() {
         });
     }
 
+    @Test
+    public void testNearCacheStatsWithReadOnlyProcessor() {
+        int mapSize = 1000;
+        String mapName = randomMapName();
+
+        Config config = getConfig();
+        config.getMapConfig(mapName).setNearCacheConfig(newNearCacheConfig().setInvalidateOnChange(false));
+
+        TestHazelcastInstanceFactory factory = createHazelcastInstanceFactory(2);
```

Comment:
I did not understand why we use 2 instances here. Did you mean to `populateNearCache` using the second map? This comment applies to all other tests

Reply:
No particular reason as such. I saw other tests were ran on more than one member hence I have used 2 instances here.